examples/sklearn/demo_grid_search_reduction_regression_sklearn.ipynb

```diff
@@ -1,718 +1,580 @@
+     ]
+    }
+   ],
+   "source": [
+    "#Install aif360\n",
+    "#Install Reductions from Fairlearn\n",
+    "#Install Low School GPA dataset\n",
+    "!pip install 'aif360[Reductions,LawSchoolGPA]'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "id": "sivw3vma71DE"
+   },
+   "outputs": [
     {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "lukGdCjv71DJ"
-      },
-      "source": [
-        "### Running metrics"
-      ]
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:root:No module named 'tensorflow': AdversarialDebiasing will be unavailable. To install, run:\n",
+      "pip install 'aif360[AdversarialDebiasing]'\n"
+     ]
     },
     {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "x9LheHu-71DK"
-      },
-      "source": [
-        "With the data in this format, we can easily train a scikit-learn model and get predictions for the test data. We drop the protective attribule columns so that they are not used in the model."
-      ]
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/Users/ektabhaskar/Desktop/GHC/AIF360/examples/sklearn/demo_grid_search_reduction_regression_sklearn.ipynb\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import sys\n",
+    "import os\n",
+    "\n",
+    "from sklearn.compose import TransformedTargetRegressor\n",
+    "from sklearn.linear_model import LinearRegression\n",
+    "from sklearn.metrics import mean_absolute_error\n",
+    "from sklearn.preprocessing import MinMaxScaler\n",
+    "current_script_path=os.path.abspath('demo_grid_search_reduction_regression_sklearn.ipynb')\n",
+    "print(current_script_path)\n",
+    "sys.path.append('..')\n",
+    "sys.path.append('/Users/ektabhaskar/Desktop/GHC/AIF360/aif360/sklearn/datasets/')\n",
+    "# from aif360.sklearn.datasets.seaphe_datasets import load_lawschool_data\n",
+    "from seaphe_datasets import load_lawschool_data\n",
+    "# aif360/sklearn/datasets/seaphe_datasets.py\n",
+    "from aif360.sklearn.inprocessing import GridSearchReduction\n",
+    "# from aif360.sklearn.metrics import difference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "uMTJ62DZ71DF"
+   },
+   "source": [
+    "### Loading data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "KgqbgiIY71DG"
+   },
+   "source": [
+    "Datasets are formatted as separate `X` (# samples x # features) and `y` (# samples x # labels) DataFrames. The index of each DataFrame contains protected attribute values per sample. Datasets may also load a `sample_weight` object to be used with certain algorithms/metrics. All of this makes it so that aif360 is compatible with scikit-learn objects.\n",
+    "\n",
+    "For example, we can easily load the law school gpa dataset from tempeh with the following line:"
```

Comment:
can you change the text to reflect that we're fetching it from SEAPHE now?

Reply:
Updated